multigeiger/multigeiger.ino

```diff
@@ -178,7 +178,7 @@ typedef struct {
 TUBETYPE tubes[] = {
   {"Radiation unknown", 0, 0.0},                            // use 0.0 conversion factor for unknown tubes, so it computes an
                                                             // "obviously-wrong" 0.0 uSv/h value rather than a confusing one.
-  // XXX FIXME: add comments about "how are the conversion factors determined / computed? sources for these values?"
+  // The conversion factors for SBM-20 and SBM-19 are taken from the datasheets (according to Jürgen)
   {"Radiation SBM-20", 20, 1/2.47},
   {"Radiation SBM-19", 19, 1/9.81888},
```

Comment:
could not find a non-russian datasheet. so same question here + url of datasheet.